Assets/SEE/Net/Network.cs

```diff
@@ -629,153 +573,201 @@ private void OnSceneUnloaded(Scene scene)
         public delegate void CallBack(bool success, string message);
 
         /// <summary>
-        /// The IP4 address, port, and protocol.
-        /// </summary>
-        private string ServerAddress => $"{ServerIP4Address}:{ServerPort} (UDP)";
-
-        /// <summary>
-        /// Starts a host process, i.e., a server and a local client.
+        /// Starts a server process.
         ///
         /// Note: This method starts a co-routine and then returns to the caller immediately.
         /// The <paramref name="callBack"/> tells the caller that the co-routine has come to
         /// an end.
         /// </summary>
         /// <param name="callBack">a callback to be called when done; its parameter will be true
         /// in case of success or otherwise false</param>
-        public void StartHost(CallBack callBack)
+        public void StartServer(CallBack callBack)
         {
-            StartCoroutine(ShutdownNetwork(InternalStartHost));
+            StartCoroutine(ShutdownNetwork(InternalStartServer));
 
-            void InternalStartHost()
+            void InternalStartServer()
             {
+                // Using an IP address of 0.0.0.0 for the server listen address will make a
+                // server or host listen on all IP addresses assigned to the local system.
+                // This can be particularly helpful if you are testing a client instance
+                // on the same system as well as one or more client instances connecting
+                // from other systems on your local area network. Another scenario is while
+                // developing and debugging you might sometimes test local client instances
+                // on the same system and sometimes test client instances running on external
+                // systems.
+                ServerIP4Address = "0.0.0.0"; //FIXME: Why this assignment?
                 Debug.Log($"Server is starting to listen at {ServerAddress}...\n");
-                Debug.Log($"Local client is trying to connect to server {ServerAddress}...\n");
                 try
                 {
-                    if (NetworkManager.Singleton.StartHost())
+                    NetworkManager.Singleton.ConnectionApprovalCallback = ApprovalCheck;
+                    if (NetworkManager.Singleton.StartServer())
                     {
-                        StartUp();
+                        InitializeGame();
+                        NetworkManager.Singleton.OnClientConnectedCallback += OnClientConnectedCallbackForServer;
+                        NetworkManager.Singleton.OnClientDisconnectCallback += OnClientDisconnectCallbackForServer;
                     }
                     else
                     {
-                        throw new CannotStartServer($"Could not start host {ServerAddress}");
+                        throw new CannotStartServer($"Could not start host {ServerAddress}.");
```

Comment:
Consistency with surrounding naming conventions:
```suggestion
                        throw new CannotStartServer($"Could not start server {ServerAddress}.");
```